incubator/kafka/templates/statefulset.yaml

```diff
@@ -158,6 +158,8 @@ spec:
               fieldPath: metadata.namespace
         - name: KAFKA_HEAP_OPTS
           value: {{ .Values.kafkaHeapOptions }}
+        - name: KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR
+          value: {{ .Values.replicas | quote }}
```

Comment:
Can you make this a template? If a user specifies something else in the values file (configurationOverrides or otherwise) it would be good if that value was set.

Reply:
@benjigoldberg Do you mean that we'll define a template in `_helpers.tpl`? Would `configurationOverrides["offsets.topic.replication.factor"]` override `replicas` value?